supabase/migrations/20240415132809_add_creation_timestamp_to_parcels.sql

```diff
@@ -0,0 +1,33 @@
+alter table "public"."parcels" add column "created_at" timestamp with time zone not null default (now() AT TIME ZONE 'utc'::text);
+
+update "public"."parcels" set "created_at"='2023-12-31 12:00:00+00';
+
+drop view if exists "public"."parcels_plus";
+
+create or replace view "public"."parcels_plus" as  SELECT parcels.primary_key AS parcel_id,
+    parcels.collection_datetime,
+    parcels.packing_date,
+    parcels.created_at,
+    packing_slots.name AS packing_slot_name,
+    packing_slots.order AS packing_slot_order,
+    parcels.voucher_number,
+    collection_centres.name AS collection_centre_name,
+    collection_centres.acronym AS collection_centre_acronym,
+    clients.primary_key AS client_id,
+    clients.full_name AS client_full_name,
+    clients.address_postcode AS client_address_postcode,
+    clients.flagged_for_attention AS client_flagged_for_attention,
+    clients.signposting_call_required AS client_signposting_call_required,
+    clients.phone_number AS client_phone_number,
+    family_count.family_count,
+    last_status.event_name AS last_status_event_name,
+    last_status.event_data AS last_status_event_data,
+    last_status."timestamp" AS last_status_timestamp,
+    last_status.workflow_order AS last_status_workflow_order
+   FROM (((((parcels
+     LEFT JOIN collection_centres ON ((parcels.collection_centre = collection_centres.primary_key)))
+     LEFT JOIN clients ON ((parcels.client_id = clients.primary_key)))
+     LEFT JOIN packing_slots ON ((parcels.packing_slot = packing_slots.primary_key)))
```

Comment:
These three used to be `JOIN` but changed to `LEFT JOIN` as we want all parcels to be in parcels_plus whether they have a corresponding collection centre, client or packing slots or not.

Reply:
is the number of entries in parcels_plus still as expected (think it should be 5000?)? I think I might have experienced problems with these different types of joins when originally making the view. But I can't remember the specifics so happy with it if it works / results in right number of parcels

Reply:
parcels_plus still has 5000 items 👍 

`LEFT_JOIN` means we retain every row from the `parcels` table in this context, so we're guaranteeing that there will be 5000 items this way.

`JOIN` means we retain rows from the `parcels` table where the `ON` condition is matched, so we may end up with less than 5000 items. However, we happen to have 5000 items anyway because every parcel has a collection centre, a client and a packing slot.